### PR TITLE
Adding content to index page

### PR DESCRIPTION
**Pull request by ManpreetM01**

### CODE REVIEW

**Comment by ManpreetM01** [comment]
Screenshot 2023-09-27 at 12 12 17

File: src/controller/jobRoleController.ts [hunk 1]
```diff
@@ -13,4 +13,7 @@ export namespace JobRoles {
 
         res.render("job-roles");
     }
+    export async function getIndex(req: Request, res: Response): Promise<void> {
```
Comment (TomBillingtonUK) [inline src/controller/jobRoleController.ts]: I would want to see this in it's own controller as it has nothing to do with job roles

File: src/router.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ const router = express.Router();
 
 router.get("/login", Auth.get)
 router.post("/login", Auth.post)
+router.get("/", JobRoles.getIndex)
```
Comment (TomBillingtonUK) [inline src/router.ts]: Should this be protected by Auth

File: views/index.html [hunk 3]
```diff
@@ -6,12 +6,10 @@
...
-    <p> Paragraph </p>
-    <h4> Heading 4 </h4>
-    <p> Paragraph </p>
-{% endblock %}
\ No newline at end of file
+    <h3> Job Roles </h3>
+    <a href="/job-roles" type="submit" class="btn btn-primary text text-white">View Job Roles</a>
+    <a href="/job-roles/matrix" type="submit" class="btn btn-primary text text-white">View Matrix</a>
+    <h3> Capabilities </h3>
+    <a href="/capabilities/create" type="submit" class="btn btn-primary text text-white">Create Capability</a>
```
Comment (TomBillingtonUK) [inline views/index.html:14]: should only be there for admin

File: src/controller/jobRoleController.ts (hunk 1 above)
Comment (ManpreetM01) [reply src/controller/jobRoleController.ts]: change made

File: views/index.html (hunk 3 above)
Comment (ManpreetM01) [reply views/index.html:14]: change made

File: src/router.ts (hunk 2 above)
Comment (ManpreetM01) [reply src/router.ts]: Yes, change made

**Review by TomBillingtonUK — APPROVED** [review]

**Review by LMBishop — APPROVED** [review]

**Pull request merged**